framework/include/problems/FEProblemBase.h

```diff
@@ -1761,6 +1761,8 @@ class FEProblemBase : public SubProblem, public Restartable
    */
   void uniformRefine();
 
+  virtual void needOldSolutionState(const unsigned int state) override;
```

Comment:
Is it not possible to get rid of this?  We should't need all the manual forwarding to displaced problem and aux+nonlin systems with the lazy init.